packages/react/src/components/Masthead/MastheadSearchSuggestion.js

```diff
@@ -44,10 +49,10 @@ const MastheadSearchSuggestion = ({
 
   return (
     <div
-      className={classNames('container-class', {
-        ['container-highlight-class']: isHighlighted,
+      className={classNames(`${prefix}--container-class`, {
+        [`${prefix}--container-highlight-class`]: isHighlighted,
```

Comment:
Does anyone know what `${prefix}--container-highlight-class` is doing? I didn't see a reference for it in the `SCSS`.

Reply:
@photodow Looks like this is a leftover from when the autosuggest was first being prototyped. It doesn't need to be there and all the HTML needed for this are already provided by the package. I'll make an issue to clean this up in the future.